kernel/kernel-defaults/src/test/resources/spark-variant-checkpoint/_delta_log/00000000000000000000.json

```diff
@@ -0,0 +1,5 @@
+{"commitInfo":{"timestamp":1723657184248,"operation":"CREATE OR REPLACE TABLE AS SELECT","operationParameters":{"partitionBy":"[]","clusterBy":"[]","description":null,"isManaged":"false","properties":"{\"delta.checkpointInterval\":\"2\"}"},"isolationLevel":"Serializable","isBlindAppend":false,"operationMetrics":{"numFiles":"2","numOutputRows":"100","numOutputBytes":"44813"},"engineInfo":"Apache-Spark/4.0.0-SNAPSHOT Delta-Lake/3.3.0-SNAPSHOT","txnId":"61d81b32-bfce-4bae-a021-ee43779c91fe"}}
```

Comment:
add a `info.txt` file that contains the commands used to generate this table. Helpful in future when we generate this table directly.